examples/deployment/docker/log_server/Dockerfile

```diff
@@ -4,6 +4,8 @@ ADD . /go/src/github.com/google/trillian
 WORKDIR /go/src/github.com/google/trillian
 
 ARG GOFLAGS=""
```

Comment:
Separately, at should add an ENV directive for $GOFLAGS too, since it'll only apply to the `go get` right now (ARG drops out of scope between layers).

Reply:
That sounds like a separate PR